src/containers/Header.js

```diff
@@ -5,7 +5,8 @@ export default class Header extends Component {
   render() {
     return(
       <section id="header">
-        <p id="current-local-forecast">Current forecast for <span id="city-name">Mikes Butt</span></p><p id="view-extended-forecast">Lots and lots of poopoo</p>
+        <p id="current-local-forecast">Current forecast for <span id="city-name">Mikes Butt</span></p>
+        <p id="view-extended-forecast">Lots and lots of poo-poo</p>
```

Comment:
Super glad this is in your git history for all of your potential future employers to see.